src/components/Carousel.js

```diff
@@ -614,6 +614,7 @@ export default class Carousel extends Component {
    * @return {ReactElement} element
    */
   renderArrowLeft = () => {
```

Comment:
could you fix the typo in commit message `replace value with getCurretnValue and slides with getChildren` -> `replace value with getCurrentValue and slides with getChildren` (getCurretnValue -> getCurrentValue) and force push?